app/repositories/sift/ApplicationSiftRepository.scala

```diff
@@ -140,6 +142,27 @@ class ApplicationSiftMongoRepository(
     bsonCollection.find(query).cursor[Candidate]().collect[List]()
   }
 
+  def findAllResults: Future[Seq[SiftPhaseReportItem]] = {
+    val query = BSONDocument.empty
```

Comment:
Does this purposefully return people with no results? even people who haven't entered sift yet? This could perhaps be accomplished with a query that doesn't make the database work as hard (query: testGroups.$phaseName.evaluation.result { $exists: true })